source/guide_gitea.rst

```diff
@@ -200,112 +191,44 @@ For more information about the possibilities and configuration options see the G
   MAILER_TYPE = sendmail
   FROM        = isabell@uber.space
 
-
-Gitea using external renderer (optional)
-----------------------------------------
-
-We can install an extra `external rendering <https://docs.gitea.io/en-us/external-renderers/>`_ extension AsciiDoc.
-
-.. code-block:: console
-
-  [isabell@stardust ~]$ gem install asciidoctor
-  Fetching asciidoctor-2.0.10.gem
-  WARNING:  You don't have /home/isabell/.gem/ruby/2.7.0/bin in your PATH,
-  	  gem executables will not run.
-  Successfully installed asciidoctor-2.0.10
-  1 gem installed
-  [isabell@stardust ~]$
-
-.. note:: Don't be irritated by the warning that the bin folder isn't in the path. Uberspace is taking care of it. You can check with ``[isabell@stardust ~]$ which asciidoctor``.
-
-Now we have to append the config file ``~/gitea/custom/conf/app.ini`` with:
-
-.. code-block:: ini
-
-  [markup.asciidoc]
-  ENABLED = true
-  FILE_EXTENSIONS = .adoc,.asciidoc
-  RENDER_COMMAND = "asciidoctor -e -a leveloffset=-1 --out-file=- -"
-  IS_INPUT_FILE = false
+  [repository]
+  DEFAULT_BRANCH = main
```

Comment:
thanks for doing these updates, but still, I think this configuration part is too much confusing and we should reduce it somehow more. Its hard to explain what I have in mind so I will try to commit it by myself :-)